source/_integrations/sma.markdown

```diff
@@ -17,9 +17,14 @@ ha_integration_type: integration
 
 The SMA Solar integration will poll a [SMA](http://www.sma-solar.com/) [(US)](https://www.sma-america.com/) solar inverter, energy meter or battery inverter and present the values as sensors in Home Assistant.
 
-The integration uses the web interface of the device. Before you start, make sure you are able to connect to the solar inverter from your favorite web browser.
 
 {% include integrations/config_flow.md %}
+## Supported Devices
+Not all inverters, energy meters and batteries from SMA are supported.
+
+The integration uses the web interface (called webconnect) of the device. Before you start, make sure you are able to connect to the solar inverter from your favorite web browser. Devices with the ennexos operating system (e.g. Tripower X series) are not supported. Alternatives for accessing these data are Modbus or [Sunspec](https://github.com/CJNE/ha-sunspec).
```

Comment:
We can not recommend or list custom integrations. We don't support or recommend using them.